odl/phantom/noise.py

```diff
@@ -25,17 +25,62 @@
 import numpy as np
 
 
-__all__ = ('white_noise',)
+__all__ = ('white_noise', 'poisson_noise')
 
 
 def white_noise(space):
     """Standard gaussian noise in space, pointwise N(0, 1).
 
+    Parameters
+    ----------
+    space : `FnBase`
+        The space that the noise should be created in
```

Comment:
Convoluted sentence - reads easier as `The space in which the noise should be created.`
BTW, why "should be"? Why not "is"?